components/header/Navbar.tsx

```diff
@@ -67,25 +85,14 @@ function Navbar(
 
   // Desktop header
   return (
```

Comment:
No caso do header desktop, ele pode ser o ELSE do seu ternário para o retorno do componente, ficando dessa forma:
```suggestion

  // Desktop header
  ...
  ) : (
  conteúdo do componente de header desktop
  )
```